src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml

```diff
@@ -215,20 +221,22 @@
                     <ui:msg description="Features">Features</ui:msg>
                 </h2>
                 <div class="{res.grid.row} {res.style.clearfix} {res.style.wrapper}">
-                    <div class="{res.grid.col} {res.grid.col_1_4} {res.grid.col_t_1_3} {res.grid.col_t_push_0} {res.grid.col_m_full} {res.grid.col_m_push_0}">
+                    <div class="{res.grid.col} {res.grid.col_1_4} {res.grid.col_t_1_3} {res.grid.col_t_push_0}
+                    {res.grid.col_m_full} {res.grid.col_m_push_0}">
```

Comment:
@LarryMatte why do you have `col_t_push_0` and `col_m_push_0` here? It looks like bad code to me, as there is no push for desktop and push_0 is supposed to overwrite a previous push ... (this comment goes for next items having the same pattern)